### ISSUE TITLE
Investigate move to Container-Linux?

### ISSUE THREAD

**Issue by andersla** [kubenow/KubeNow#402]
For easier OS maintenance and security upgrades, Investigate moving KubeNow image from current Ubuntu 16 to Container Linux (now supported by Kubeadm - https://kubernetes.io/docs/tasks/tools/install-kubeadm/)

**Comment by mcapuccini** [comment]
Good idea! Perhaps we could pack a single Docker image that would then work on any cloud provider.
